scipy/stats/_continuous_distns.py

```diff
@@ -4089,6 +4089,63 @@ def _entropy(self):
 semicircular = semicircular_gen(a=-1.0, b=1.0, name="semicircular")
 
 
+class skew_normal_gen(rv_continuous):
+    """A skew-normal random variable
+    
+    %(before_notes)s
+    
+    Notes
+    -----
+    The pdf is
+
+    skewnormal.pdf(x, a) = 2*norm.pdf(x)*norm.cdf(ax)
+    
+    `skewnormal` takes ``a`` as a skewness parameter
+    When a=0 the distribution is identical to a normal.
+    rvs implements the method of 
+   
+    %(after_notes)s
+    
+    %(example)s
+    
+    
+    References
+    ----------
+    
+    A. Azzalini and A. Capitanio (1999). Statistical applications of the multivariate skew-normal distribution. J. Roy. Statist. Soc., B 61, 579-602.
+   
+    http://azzalini.stat.unipd.it/SN/faq-r.html
+    """
+
+    def _argcheck(self, a):
+        return np.isfinite(a)
+        
+    def _pdf(self, x, a):
+        return 2.*_norm_pdf(x)*_norm_cdf(a*x)
+        
+    def _rvs(self, a):
+        [u0, v] = norm.rvs(size=(2, ) + self._size)
```

Comment:
Making the l.h.s. a list seems confusing and unneeded. Any particular reason you've decided to use a list, not tuple (u, v) ?